src/Tasks/WriteCodeFragment.cs

```diff
@@ -170,9 +185,15 @@ private string GenerateCode(out string extension)
             }
 
             // For convenience, bring in the namespaces, where many assembly attributes lie
-            foreach (string name in NamespaceImports)
+            // This will be skipped for only a single situation:
+            // - When Language is "C#" and sets ImplicitUsings to "enable".
+            // Checking only ImplicitUsings is unknown if it works for Visual Basic.
+            if (ImplicitUsings is "disable" or "false")
             {
-                globalNamespace.Imports.Add(new CodeNamespaceImport(name));
+                foreach (string name in NamespaceImports)
```

Comment:
Not including System makes sense to me, but don't we still need System.Reflection?

Reply:
Yes, because the .NET SDK does not implicitly set it yet (although I think it should).